server/app/com/xsn/explorer/gcs/GolombEncoding.scala

```diff
@@ -21,16 +21,20 @@ class GolombEncoding(p: Int, m: Int, key: SipHashKey) {
    * Encodes the given word set.
    */
   def encode(words: Set[String]): Option[GolombCodedSet] = {
-    if (words.isEmpty) {
+    encodeData(words.map(_.getBytes))
+  }
+
+  def encodeData(data: Set[Array[Byte]]): Option[GolombCodedSet] = {
```

Comment:
The reason that the filter may not be computed was because we weren't able to retrieve the genesis block transactions, which means, it would have been always empty, now that we can get its transactions, there shouldn't be a need to have an optional result.